std/algorithm/setops.d

```diff
@@ -611,6 +593,27 @@ void largestPartialIntersection
             sorted);
 }
 
+///
```

Comment:
Sure, you can also put text in the comment for the unit test and it will appear before the example code.